unstable/wlr-export-dmabuf-unstable-v1.xml

```diff
@@ -0,0 +1,230 @@
+      once per layer. The "dma_plane" callback shall only be called after
+      the "dma_layer" callback corresponding to the layer the plane belongs
+      to has been called. Finally, the "ready" event is called to indicate that
+      all the data has been made available for readout, as well as the time
+      at which presentation happened at.
+      The ownership of the frame is passed to the client, who's responsible for
+      destroying it via the "destroy" event once finished.
+      The data the API describes has been based off of what
+      VASurfaceAttribExternalBuffers contains.
+      All frames are read-only and may not be written into or altered.
+    </description>
+
+    <enum name="frame_flags">
+      <description summary="frame flags">
+        Special flags that must be respected by the client.
+        Transient frames indicate short lifetime frames (such as swapchain
+        images from external clients). Clients are advised to copy them and do
+        all processing outside of the "ready" event.
+      </description>
+      <entry name="transient" value="1" since="1"/>
+    </enum>
+
+    <event name="frame">
+      <description summary="main callback">
+        Main callback supplying the client with information about the frame,
+        as well as an object to serve as context for destruction. Always called
+        first before any other events.
+
+        The "transform" argument describes the orientation needed to be applied
+        to correctly orient the buffer. For example, a buffer rotated by 90
+        degrees will have a value of "3" here, corresponding to the need to
+        apply a 270 degree transpose to correctly present the buffer.
+      </description>
+      <arg name="width" type="uint"
+           summary="frame width, scaling factor included"/>
+      <arg name="height" type="uint"
+           summary="frame height, scaling factor included"/>
+      <arg name="scale" type="uint"
+           summary="current scaling factor of the client"/>
+      <arg name="transform" type="uint"
+           summary="transform needed to correct the orientation, has the same
+                    values as wl_output::transform"/>
+      <arg name="flags" type="uint"
+           summary="flags which indicate properties (invert, interlacing),
+                    has the same values as zwp_linux_buffer_params_v1:flags"/>
+      <arg name="frame_flags" type="uint" enum="frame_flags"
+           summary="indicates special frame features"/>
+      <arg name="mod_high" type="uint"
+           summary="drm format modifier, high"/>
+      <arg name="mod_low" type="uint"
+           summary="drm format modifier, low"/>
+      <arg name="num_objects" type="uint"
+           summary="indicates how many objects (FDs) the frame has (max 4)"/>
+      <arg name="num_layers" type="uint"
+           summary="indicates how many layers the frame has (max 4)"/>
+    </event>
+    <event name="dma_object">
```

Comment:
Again, this gives symbols like `zwlr_export_dmabuf_frame_v1_send_dma_object`, with `dma` appearing twice